features/smamodbus.py

```diff
@@ -374,25 +374,33 @@
         9314: 'PlugwiseCircle',
         9315: 'PlugwiseSting',
         9316: 'SCS-1000',
-        9317: 'SB 5400TL-JP-22'
+        9317: 'SB 5400TL-JP-22',
+        9324: 'SBS1.5-1VL-10',
+        9325: 'SBS2.0-1VL-10',
+        9326: 'SBS2.5-1VL-10',
+        9344: 'STP4.0-3AV-40',
```

Comment:
Please add:

```
9456: 'SBS3.7-10'
```
I can confirm that the modbus register and values are the same. I am happy to help testing because i am using this battery inverter if there is anything needed.

Reply:
SBS3.7-10 should be 9**3**56 according to this: https://files.sma.de/downloads/MODBUS-HTML_SBS3.7-6.0-10_GG10-V13.zip

Will add these to the list, but actual readout of battery-related values is not yet implemented.

Reply:
sorry yes, typo.